xray/src/event_loop.rs

```diff
@@ -42,7 +42,7 @@ impl EventLoop {
             is_done: false,
             crypto_buf: vec![0; 1024],
             plaintext_buf: vec![0; 1024],
-            recv_counter: 1,
+            recv_counter: 0,
```

Comment:
This makes more sense with the debug prints, or rather, doing it this way allows for better debug prints without having to do a `-1` whenever it is referenced. A `+1` is needed in a couple of places instead, but significantly fewer than the number of `-1`s that would be otherwise needed